packages/frontend/editor-ui/src/components/RunData.vue

```diff
@@ -266,14 +267,18 @@ const displayModes = computed(() => {
 		defaults.push(schemaView);
 	}
 
+	const isHtmlNode = activeNode.value?.type === HTML_NODE_TYPE;
```

Comment:
Looks like we are duplicating all this conditions. Can we have it in one place in the file and reuse them?